### DISCUSSION TITLE
Install TomoATT on Gekko

### DISCUSSION THREAD

**Discussion by JingChen-Thu** [TomoATT/TomoATT#2] [Q&A]
When I follow the instruction to install TomoATT on Gekko via this link "https://tomoatt.com/docs/GetStarted/Use_on_HPCs"

An error arose that "../build/bin/TOMOATT: /usr/local/gcc-9.3.0/lib64/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by /home/jing.chen/.conda/envs/tomoatt/lib/./././libicuuc.so.75)"

After replacing "gcc-9.3.0" with "gcc-13.1.0", everything works well. Is it possible to make a modification for the instruction? Thank you!

e.g., 

![image](https://github.com/user-attachments/assets/0e35270d-5602-4163-a071-35685d6f78f8)

![image](https://github.com/user-attachments/assets/7912fa75-05b0-459d-8b30-8f3a81911607)

**Comment by xumi1993** [comment]
Hi @mnagaso, I updated some content on TomoHP https://github.com/mnagaso/TomoHP/commit/26302c28487af624346821a859a18025de4320a4, but an error occurred during deployment. We may need your help. 😄 
![Screenshot 2024-10-22 at 9 59 48 AM](https://github.com/user-attachments/assets/9e373a48-8fbb-4282-ae47-e5148d2c5c9b)

**Comment by mnagaso** [comment]
Hi @JingChen-Thu, @xumi1993 

Thanks for updating the installation steps for Gekko!
As you've already found, it seems that Vercel deployment requires the access to the project on the Vercel server.

I tried to allow your access request but as we are using free plan, it is not possible to add another person. So I would like to ask you to create PR for each update of the document (on TomoHP repo.) so that I can find it and deploy it from my side.

On the other hand, you can check the modified document on the local machine with
`npm install` then `npm run dev`

the Tomoatt.com opens with `localhost:3000` and documentation site with `localhost:4000/docs` 

change of the port number is necessary only for the local test. Also, the top link `Documents` on the Tomoatt.com does not work on local test, which is because the routing method on Vercel server and local npm server differs.

Thanks again!
